src/Components/ArticleList.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from "react"
+import ArticleCard from './ArticleCard'
+import { fetchArticlesByTopics } from "./Api"
+import '../App.css'
+
+const ArticleList = ({topics}) => {
+
+    const [articles, setArticles] = useState([])
+    const [selectedTopic, setSelectedTopic] = useState("all")
+
+    useEffect(() => {
+        fetchArticlesByTopics(selectedTopic)
+          .then((fetchedArticles) => {
+            setArticles(fetchedArticles)
+          })
+          .catch((error) => {
+            console.error('Error fetching items:', error)
+          })
+      }, [selectedTopic])
+  
+    function handleChange(event) {
+      setSelectedTopic(event.target.value)
+    }
+  
+  
+  
+      return (
+          <>
+          <h2>
+              Browse our Articles 
+          </h2>
+          <label htmlFor="topics-dropdown">Filter by Topic: </label>
+          <select 
+          id='topics-dropdown'
+          value={selectedTopic} 
+          onChange={handleChange}
+        >
+          <option value="all">All</option>
+          {topics.map((topic) => (
+            <option key={topic.slug} value={topic.slug}>
```

Comment:
Viewing a list of articles filtered by topic is _ticket 10_ of this sprint, so none of this code should be implemented as part of ticket 4 

Try to stick to the brief for the ticket you are currently working on, going off track for a ticket can get you into trouble with your Lead Developers / Product Managers when you are working as a developer in the industry